great_ape_safe/ape_api/citadel.py

```diff
@@ -108,3 +112,13 @@ def get_pricing_oracle(self, asset):
     def get_asset_cap(self, asset):
         funding_pool = self.get_funding_contract(asset)
         return funding_pool.getFundingParams()[5]
+
+    def distribute_yield(self, asset, mantissa):
+        assert mantissa > 0
+        asset = interface.ERC20(asset, owner=self.safe.account)
```

Comment:
imo better to use `Contract/safe.contract` opposed to `interface.ERC20` since `asset` is variable